src/App.tsx

```diff
@@ -48,6 +48,7 @@ const App: FC = () => {
     if (CLOUD && isFlagEnabled('rudderstackReporting')) {
       load(RUDDERSTACK_WRITE_KEY, RUDDERSTACK_DATA_PLANE_URL)
     }
+    setAutoFreeze(process.env.NODE_ENV !== 'production')
```

Comment:
added this because immer changed the default freezing behavior and recommends running this to revert back to the original behavior: https://github.com/immerjs/immer/releases/tag/v8.0.0